tests/resources/modules/d7/woot/Command/.gitignore

```diff
@@ -0,0 +1 @@
+WootCommands.php
```

Comment:
Rather than make two copies of the same 'WootCommands', my scripts copy or symlink the Drupal 8 version into the Drupal 7 test module directory.  I don't want this test artifact to be accidentally committed. It might also be a good idea to just erase the duplicate after the test runs.